### PR TITLE
Add Events as a property of TracePayload

### PR DESCRIPTION
**Pull request by Mmarzex**

### CODE REVIEW

File: .github/workflows/node-validate.yml [hunk 1]
```diff
@@ -252,11 +252,14 @@ jobs:
       - name: Buf Setup
         if: steps.pathChanges.outputs.sdkSchema == 'true' || steps.pathChanges.outputs.proto == 'true'
         uses: bufbuild/buf-setup-action@v1
+        with:
+          github_token: ${{ github.token }}
```
Comment (medikoo) [inline .github/workflows/node-validate.yml:256]: Why these changes are needed?

Comment (Mmarzex) [reply .github/workflows/node-validate.yml:256]: `buf` is rate limited. So if you pass in the github_token it uses that to authenticate/reset the rate limit. This is the safer method then using one of their authentication tokens in a PR workflow as that token would allow pushes into our buf repo which we do not want.

**Review by medikoo — APPROVED** [review]
👍
